avail-server/src/main/java/com/avail/server/io/WebSocketAdapter.kt

```diff
@@ -1244,6 +1296,7 @@ class WebSocketAdapter @Throws(IOException::class) constructor(
 						}
 						else
 						{
+							channel.heartbeat.receiveHeartbeat()
 							sendPong(
 								strongChannel, bytes.toByteArray(), null, null)
```

Comment:
Looks like we might want to give `sendPong` default arguments of `null` for the last two parameters.